wapitiCore/parsers/commandline.py

```diff
@@ -155,7 +155,7 @@ def parse_args():
         choices=["basic", "digest", "ntlm"]
     )
 
-    # This option is deprecated
+    # This option is deprecated
```

Comment:
?

Reply:
It was a non breaking space, PEP8 doesn't like it.